test/e2e/util/util.go

```diff
@@ -14,6 +13,11 @@ import (
 	. "github.com/onsi/gomega"
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/openshift/hypershift/test/e2e/util/cluster"
+	e2ecluster "github.com/openshift/hypershift/test/e2e/util/cluster/aws"
+	"github.com/openshift/hypershift/test/e2e/util/cluster/none"
```

Comment:
nit: Can we drop/have consistent alias for these two?

Reply:
Done. Since there is already an `aws` import I named them `awscluster` and `nonecluster`.